lnst/Recipes/ENRT/UseVfsMixin.py

```diff
@@ -56,7 +63,7 @@ def generate_test_wide_description(self, config):
 
         if self.params.use_vfs:
             description += [
-                f"Using vf device {vf_dev.name} of pf {sriov_devices.phys_dev.name} for DeviceReq {host.hostid}.{vf_dev._id}"
+                f"Using vf device {vf_dev.name} of pf {sriov_devices.phys_dev.name} for DeviceReq {host.hostid}.{vf_dev._id} trusted={self.params.vf_trust}"
```

Comment:
this directly prints `self.params`... is there maybe also a `getter` for the `Device.vf_trust` value that should be used instead to reflect the real state of the device?

Reply:
There's no getter atm. The "property" is implemented through a method. Maybe I could change this to property.